frontend/src/general/SnackbarMessage.jsx

```diff
@@ -0,0 +1,166 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Snackbar, SnackbarContent, IconButton, Box } from "@material-ui/core";
+import { CheckCircle, Error, Close } from "@material-ui/icons";
+import { withRouter } from "react-router";
+
+let openSnackbarFn;
```

Comment:
You can rename this to openSnackbar without getting name conflicts since the other openSnackbar is a member function of the SnackbarMessage class.